apps/services/schemas.py

```diff
@@ -13,6 +13,78 @@
 )
 
 
+speciality = {51: 'Економіка', 121: 'Інженерія програмного забезпечення', 122: "Комп'ютерні науки", 124: 'Системний аналіз',
+              125: 'Кібербезпека', 126: 'Інформаційні системи та технології', 186: 'Видавництво та поліграфія',
+              53: 'Психологія', 81: 'Право, освітня програма', 232: 'Соціальне забезпечення',
+              281: 'Публічне управління та адміністрування', 73: 'Менеджмент', 75: 'Маркетинг',
+              22: 'Дизайн', 72: 'Фінанси, банківська справа та страхування', 71: 'Облік і оподаткування',
+              11: '"Освітні, педагогічні науки"', 52: 'Політологія', 61: 'Журналістика',
+              291:'Міжнародні відносини, суспільні комунікації та регіональні студії', 292: 'Міжнародні економічні відносини',
+              76: '"Підприємництво, торгівля та біржова діяльність"', 241: 'Готельно-ресторанна справа', 242: 'Туризм'}
+
+
+class CreateCustomHostelAccommodationIn(BaseInSchema):
```

Comment:
RequestForHostelAccommodationIn is better
Why did you write the word "custom" everywhere?

Reply:
ok